enterprise/frontend/src/embedding-sdk/CHANGELOG.md

```diff
@@ -71,7 +71,7 @@
 
 ### Features
 
-* **sdk:** `defineEmbeddingSdkConfig` to make typing easier ([#48879](https://github.com/metabase/metabase/issues/48879)) ([aa72eb8](https://github.com/metabase/metabase/commit/aa72eb83aebd2adc3dbd700d28f1b7d18b2e0987))
+* **sdk:** `defineEmbeddingSdkAuthConfig` to make typing easier ([#48879](https://github.com/metabase/metabase/issues/48879)) ([aa72eb8](https://github.com/metabase/metabase/commit/aa72eb83aebd2adc3dbd700d28f1b7d18b2e0987))
```

Comment:
Hmmm. This is the changelog, I don't think we should modify this. We should keep this for historical reason, and we'll have a new entry after this PR land that we change this to whatever the name we'll choose.

Reply:
Thanks for catching this! I must have forgotten to exclude this from find-and-replace.